cypress/integration/QuestionForm.spec.js

```diff
@@ -0,0 +1,271 @@
+/// <reference types="cypress" />
+
+context('QuestionForm', () => {
+  beforeEach(() => {
+    cy.visit('/')
+  })
+
+  it('has the right App title', () => {
+    cy.title().should('include', 'hello')
+  })
+
+  it('has a headline', () => {
+    cy.get('h1').should('have.length', 1)
+  })
+
+  it('has a button', () => {
+    cy.get('button').contains('Zusammenfassung anzeigen').click()
+  })
```

Comment:
why do you click the button in the test?

Reply:
Hmm good question - will remove it!